### PR TITLE
chore: add meta build info to the website

### PR DESCRIPTION
**Pull request by Wxh16144**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [x] Other (about what?)

### 💡 Background and solution

image

[image
](https://private-user-images.githubusercontent.com/32004925/329165926-066ba02f-a7e5-448e-a7f3-43e61450a2f8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUyNDEzNzgsIm5iZiI6MTcxNTI0MTA3OCwicGF0aCI6Ii8zMjAwNDkyNS8zMjkxNjU5MjYtMDY2YmEwMmYtYTdlNS00NDhlLWE3ZjMtNDNlNjE0NTBhMmY4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA5VDA3NTExOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA5Yjc1Y2VjYTkyYzc2ZWIzZjUzMDU0ZGYyNWQyMjk0NDBkNDIyYmUzZWU5ZGM1MzEyM2ViZDEyY2MxNGMyNjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qPTZ5DiLMcrAFsqwT7PuTXyQ7-0lma9avqhsYv47R3Y)

不确定是否需要这样做，但是有一个展示信息总是好的，欢迎讨论~

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     --      |
| 🇨🇳 Chinese |      --     |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: .dumirc.ts [hunk 1]
```diff
@@ -40,7 +40,12 @@ export default defineConfig({
   },
   extraRehypePlugins: [rehypeAntd],
   extraRemarkPlugins: [remarkAntd],
-  metas: [{ name: 'theme-color', content: '#1677ff' }],
+  metas: [
+    { name: 'theme-color', content: '#1677ff' },
+    { name: 'build-time', content: Date.now().toString() },
+    // https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables
+    process.env.GITHUB_SHA && { name: 'build-hash', content: process.env.GITHUB_SHA },
+  ].filter(Boolean) as ReturnType<typeof defineConfig>['metas'],
```
Comment (MadCcc) [inline .dumirc.ts:48]: 这个 filter 不如反映到 content 里面去，这个防御编程感觉没啥必要，如果 sha 有问题也应该反映在产物里。

Comment (Wxh16144) [reply .dumirc.ts:48]: 噢噢，那上面那个时间 用时间戳还是format 一下呢

image

Comment (Wxh16144) [reply .dumirc.ts:48]: > 这个 filter 不如反映到 content 里面去，这个防御编程感觉没啥必要，如果 sha 有问题也应该反映在产物里。

有一个考量是，开发者本地构建就没有这个环境变量

**Comment by Wxh16144** [comment]
可直接在 antd 站点的控制台输入以下代码审查：

```js
Array.from(document.querySelectorAll('meta[name^=build]'))
  .reduce(
    (acc, meta) => ({ ...acc, [meta.name]: meta.content }),
    {}
  );
```

**Review by zombieJ — APPROVED** [review]

**Pull request merged**